e2e/test/scenarios/admin/i18n/content-translation/column-names.cy.spec.ts

```diff
@@ -45,29 +45,40 @@ describe("scenarios > admin > localization > content translation of column names
           });
         });
 
-        it("when locale is English, column names are NOT localized in column headers", () => {
-          H.snapshot("translations-uploaded");
-          cy.signInAsNormalUser();
-          H.visitQuestion(productsQuestionId);
-          cy.findByTestId("table-header").within(() => {
-            germanFieldNames.forEach((row) => {
-              cy.findByText(row.msgid).should("be.visible");
-              cy.findByText(row.msgstr).should("not.exist");
+        describe("when locale is English, column names are NOT localized in", () => {
+          beforeEach(() => {
+            H.restore("translations-uploaded" as any);
+            cy.signInAsNormalUser();
+          });
+
+          it("column headers", () => {
+            H.visitQuestion(productsQuestionId);
+            cy.findByTestId("table-header").within(() => {
+              germanFieldNames.forEach((row) => {
+                cy.findByText(row.msgid).should("be.visible");
+                cy.findByText(row.msgstr).should("not.exist");
+              });
             });
           });
         });
 
-        it("when locale is German, column names ARE localized in column headers", () => {
-          H.snapshot("translations-uploaded");
-          cy.request("PUT", `/api/user/${NORMAL_USER_ID}`, {
-            locale: "de",
+        describe("when locale is German, column names ARE localized in", () => {
+          beforeEach(() => {
+            cy.signInAsAdmin();
+            H.restore("translations-uploaded" as any);
```

Comment:
Is this using the snapshot generated in the other suite? Do we have a way of guaranteeing that it will exist?

Reply:
I fixed this - this "translations-uploaded" snapshot is now generated/restored in this suite